src/routes/navigation/rootNavigator.tsx

```diff
@@ -1,14 +1,47 @@
-import React from "react";
-import {NavigationContainer} from '@react-navigation/native';
+import React, { useEffect, useState } from "react";
+import { NavigationContainer } from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
 import TabNavigator from "./TabNavigator";
+import SplashView from "../../views/SplashView";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+const SplashScreenStack = createNativeStackNavigator();
 
 const RootNavigator = (): React.ReactElement => {
+  const [isAppFirstLaunch, setIsAppFirstLaunch] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      // AsyncStorage.clear();
+      const appData = await AsyncStorage.getItem("isAppFirstLaunch");
+      if (appData == null) {
```

Comment:
`AppData` is not a significant name, would be better something like `isAppFirstLaunch`

Reply:
Use triple `===` instead of only two